Makefile

```diff
@@ -35,3 +55,21 @@ unit-test:
 
 build: $(SOURCES)
 	go build $(GOBUILDFLAGS) -ldflags="$(GOLDFLAGS)" -o ./update-service-operator ./
+
+# Generate manifests e.g. CRD, RBAC etc.
+manifests:
+	controller-gen $(CRD_OPTIONS) rbac:roleName=updateservice-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+
+# Generate bundle manifests
+gen-for-bundle:
+	operator-sdk generate kustomize manifests -q
```

Comment:
From [your topic post][1]:

> `gen-for-bundle` generates kustomize bases and a kustomization.yaml used to generate the operator bundle. This was originally part of the `bundle` recipe as generated by the operator-sdk scaffolding. I have broken it out under the assumption that we would run this recipe and check-in the resulting config/manifests directory and contents to the git repo where they would be used by official build tools which actually generate the operator bundle.

I'm fine with that if we have access to `kustomize` and `operator-sdk` within the build environment, otherwise all of `bundle` has to be run by developers and committed to Git, right?  I don't see a point in diverging from [the scaffolded recipe][2] unless we can defer some of the logic to the build environment.

[1]: https://github.com/openshift/cincinnati-operator/pull/93#issue-529862615
[2]: https://github.com/openshift/cincinnati-operator/blob/0731f08b2fbd3989b4642ff55c6a216e97d4c990/Makefile#L109-L115

Reply:
This is really TBD until what we know what we need to provide to the official build tools vs. what the official build tools do and produce. So much of what I have here is just the way I configured things to help me understand each step.

Reply:
Added dependencies to `gen-for-bundle` on `manifest kustomize`